# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/gl/GLContext.cpp
# Commit: 898ed02804fe
# Full Hash: 898ed02804fefc9786112f135b9621cc444e6860
# Author: Jean-Yves Avenard <jyavenard@mozilla.com>
# Date: 2019-05-23 15:52:15
# Regressor Bug: 1550422
# File Overlap Count: 1
# Description:
#   Bug 1550422 - P15. Move Skip and Once gfxPrefs to StaticPrefs. r=jrmuizel
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D31259
# ==============================================================================

diff -r e1b7abc99ae9 -r 898ed02804fe gfx/gl/GLContext.cpp
--- a/gfx/gl/GLContext.cpp	Wed May 22 12:44:04 2019 +0000
+++ b/gfx/gl/GLContext.cpp	Wed May 22 22:27:37 2019 +0000
@@ -34,7 +34,7 @@
 #include "SharedSurfaceGL.h"
 #include "GfxTexturesReporter.h"
 #include "gfx2DGlue.h"
-#include "gfxPrefs.h"
+#include "mozilla/StaticPrefs.h"
 #include "mozilla/IntegerPrintfMacros.h"
 #include "mozilla/gfx/Logging.h"
 
@@ -274,7 +274,7 @@
       mDebugFlags(ChooseDebugFlags(flags)),
       mSharedContext(sharedContext),
       mCaps(caps),
-      mWorkAroundDriverBugs(gfxPrefs::WorkAroundDriverBugs()) {
+      mWorkAroundDriverBugs(StaticPrefs::WorkAroundDriverBugs()) {
   mOwningThreadId = PlatformThread::CurrentId();
   MOZ_ALWAYS_TRUE(sCurrentContext.init());
   sCurrentContext.set(0);